target_redshift/db_sync.py

```diff
@@ -688,20 +710,20 @@ def update_columns(self):
             self.table_cache = self.get_table_columns(filter_schemas=[self.schema_name])
 
     def drop_column(self, column_name, stream):
-        drop_column = "ALTER TABLE '{}' DROP COLUMN {}".format(self.table_name(stream, is_stage=False), column_name)
+        drop_column = "ALTER TABLE {} DROP COLUMN {}".format(self.table_name(stream, is_stage=False), column_name)
```

Comment:
Why remove the quotes? Will this still work if the table name has reserved chars in it?

Reply:
🤔 Ahh I think my IDE linter did this on me. It looks like it works fine without but to be safe I put them back.